### PR TITLE
Work popup

### PR DESCRIPTION
**Pull request by JauJau088**
In this PR, I and my coding partner @GedeonTS, worked together to implement the work popup feature.

Since the design of mobile and desktop versions are quite different, our approach is to have 2 cards, one for mobile and one for desktop. After that, we populate the content from data using js, according to the screen size of the browser. So, only one card will be shown depending on the screen size.

A code review is very much appreciated.

Thank you :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -242,10 +248,72 @@ <h3 class="text-green-dark">Multi-Post Stories Gain+Glory</h3>
...
+                  </button>
+                </div>
+              </div>
+              <ul class="work-tag work-tag-popup">
+                <li>dummy-text</li>
+              </ul>
+              <p class="text-blue-2 text-popup-desktop">dummy-text</p>
+            </div>
+          </div>
+        </div>
```
Comment (Whoistolu) [inline index.html]: - [ ] [REQUIRED] Hello @JauJau088 , You have done an incredible job 👍🏽 However, you need to ensure that the project section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). Kindly remove the created HTML code for the projects list.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @JauJau088 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (JauJau088) [reply index.html]: Thanks for the feedback. Updated!

File: styles/2-middle.css [hunk 2]
```diff
@@ -79,6 +80,67 @@ main {
...
+  height: 100vh;
+  padding: 20px 16px;
+}
+
+.work-popup-bg {
+  position: relative;
+  background-color: white;
+  max-width: 450px;
+  border-radius: 16px;
+}
```
Comment (phelian23) [inline styles/2-middle.css:102]: - [ ] [Optional] In the pop-up window, I can only see the image and the close button, I had to zoom out to 50% before I can see the whole pop-up, and it's not scrollable. Kindly fix this. you can make it scrollable or you adjust the image size so all the project details can be seen.
Capture

**Review by phelian23 — APPROVED** [review]
Hi @Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**